platform/ext/target/nordic_nrf/common/core/native_drivers/spu.h

```diff
@@ -40,14 +40,14 @@ void spu_enable_interrupts(void);
 void spu_clear_events(void);
 
 /**
- * \brief Reset all memory regions to being Secure
+ * \brief Reset TF-M memory regions to being Secure.
  *
- * Reset all (Flash or SRAM) memory regions to being Secure
- * and have default (i.e. Read-Write-Execute allow) access policy
+ * Reset all (Flash or SRAM) memory regions that have not been locked by any bootloader
+ * to be Secure and have default (i.e. Read-Write-Execute allow) access policy.
```

Comment:
```suggestion
 * Reset all (Flash or SRAM) memory region permissions that have not already been locked by a bootloader.
```

It sounds like a bootloader could be locking things as non-secure with the previous wording.